rules.module

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
+ * Allows site administrators to modify configuration.
```

Comment:
That does not really fit for Rules? Should be something like "Hook implementations for the Rules module."